README.md

```diff
@@ -1,2 +1,42 @@
-# turbo-rotary-phone
-El "Turbo-Rotary-Phone" es un teléfono que fusiona la nostalgia del pasado con la tecnología moderna. Su diseño compacto incluye un dial rotatorio clásico pero integrado con características avanzadas como Bluetooth y Wi-Fi. La función turbo permite una marcación rápida y una experiencia de usuario intuitiva, redefiniendo la comunicación.
```

Comment:
<  turbo-rotary-phone  >  issue error

Reply:
<  turbo-rotary-phone  >  error  this line-2, I need help with making it easier for me